### PR TITLE
Feature/gallery-menu

### PR DESCRIPTION
**Pull request by Atanas-F-Nikolov**
- create right navigation menu for gallery components.
- modify the grid for desktop, tablet & mobile view.
- fix desktop typography

### screenshots:
- closed menu
image

- opened menu
image

### CODE REVIEW

File: src/Header.tsx [hunk 1]
```diff
@@ -1,12 +1,32 @@
...
+
+export default class Header extends React.Component<object, HeaderState> {
+  constructor(props: object) {
+    super(props);
+
+    this.state = {
+      isOpen: false,
+    };
+
+    this.toggleMenu = this.toggleMenu.bind(this);
```
Comment (kblomster) [inline src/Header.tsx]: Did it really not work without this?

File: src/assets/styles/gallery-components/_menu.scss [hunk 2]
```diff
@@ -33,10 +67,13 @@ $block: 'menu';
...
-
-// react component <NavLink/> auto adds .active class to the class of active url
+} // react component <NavLink/> auto adds .active class to the class of active url
 .menu__list-item--link.active {
-  font-weight: 900;
+  font-weight: $font-weight-heavy;
   text-decoration: underline;
-}
+} 
+// @media (max-width: 991.9999px) {
```
Comment (kblomster) [inline src/assets/styles/gallery-components/_menu.scss]: Remove commented out code

File: src/assets/styles/gallery-components/_header.scss [hunk 3]
```diff
@@ -2,31 +2,47 @@ $block: 'header';
...
     margin-right: auto;
-    padding-left: 1.875rem;
-    padding-right: 1.875rem;
+    padding-left: 1.125rem;
+    padding-right: 1.125rem;
 
-    &-logo {
-      width: 159px;
-      height: 28px;
+    &-logo {
```
Comment (kblomster) [inline src/assets/styles/gallery-components/_header.scss]: No trailing whitespace, please

File: src/Header.tsx (hunk 1 above)
Comment (Atanas-F-Nikolov) [reply src/Header.tsx]: No, it doesn't work. Undefined this.

**Pull request merged**